src/UIKit/UIVibrancyEffect.cs

```diff
@@ -35,6 +35,12 @@ static public UIVibrancyEffect CreateSecondaryVibrancyEffectForNotificationCente
 		{
 			return (null as UIVibrancyEffect).GetWidgetSecondaryVibrancyEffect ();
 		}
+
+		[iOS(13, 0)]
+		static public UIVibrancyEffect CreateWidgetEffectForNotificationCenter (UIVibrancyEffectStyle vibrancyStyle)
+		{
+			return (null as UIVibrancyEffect).GetWidgetEffect (vibrancyStyle);
+		}
 #endif // IOS
```

Comment:
@spouliot something like this would do the trick ? 

Unfortunately I can't build it just now.